espnet2/train/spk_trainer.py

```diff
@@ -219,3 +224,4 @@ def validate_one_epoch(
                 nontrg_std=nontrg_std,
             )
         )
+        torch.cuda.empty_cache()
```

Comment:
Is this necessary? I think this will slow down the training.

Reply:
This is necessary at the moment after I made the validation faster. 
I confirmed that the validation speed is okay :) Thanks for the comment!

Reply:
Could you add some comments here to explain the intention? Some users might also be curious about it.

Reply:
Good comment! 
I added it to #5385 as well :)